DesignPatternAssignment-versionTwo/FactoryPatterAPI/Strategy/DisplayFareStrategy.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FactoryPatterAPI
+{
+    public class DisplayFareStrategy
+    {
+        public int IncrementFare(int currentFare, string Type)
+        {
+            Singleton.getSingletonInstance().logger("Inside DisplayFareStrategy->IncrementFare" + Environment.NewLine);
+            if (Type == "Car")
```

Comment:
for each product there should be incrementfarestrategy, 
if else isn't the correct way